src/Components/Page/Page.js

```diff
@@ -47,7 +47,7 @@ export class Page extends Component {
                   onHandleErrorClick={onHandleErrorClick}
                 />
               ))}
-            <H1>{title}</H1>
+            <H1 size={36}>{title}</H1>
```

Comment:
This is hard coding the heading to 36px, which isn't what we want since it needs to decrease to 24px on smaller devices

Reply:
`size="LARGE"` Will do this for you and apply the correct font sizes, (36px and 24px) at the correct screen widths.

Reply:
Thanks John,
That worked perfectly.